CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+v1.0.0 NOT RELEASED
+============
+
+* introduced chromehtml2pdf renderer engine
+* removed support for wkhtmltopdf renderer engine
```

Comment:
Why did we remove support for wkhtmltopdf engine? I think both engines should be supported.

Reply:
there are some unresolved issues, which will be fixed later
please switch the release version to 0.1.0 though (1.0.0 will contain both chrome and wkthml2pdf)